src/adapter/src/coord/sequencer/cluster.rs

```diff
@@ -540,21 +542,21 @@ impl Coordinator {
             .await;
     }
 
-    /// When this is called by the automated cluster scheduling, `scheduling_decision_reason` should
-    /// contain information on why is a cluster being turned On/Off. It will be forwarded to the
-    /// `details` field of the audit log event that records creating or dropping replicas.
     pub async fn sequence_alter_cluster_managed_to_managed(
         &mut self,
         session: Option<&Session>,
         cluster_id: ClusterId,
         config: &ClusterVariantManaged,
         new_config: ClusterVariantManaged,
         reason: ReplicaCreateDropReason,
-    ) -> Result<(), AdapterError> {
+        strategy: AlterClusterPlanStrategy,
+    ) -> Result<bool, AdapterError> {
```

Comment:
Using bools as fn params or return values is lightly fraught (a google search has many opinions about why). If we really really need to, then the function must be clearly documented with exactly what the boolean means and how it must be used.